cmd/sandbox-ce/cmd/web.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+	"github.com/smacker/superset-compose/cmd/sandbox-ce/compose"
+)
+
+type webCmd struct {
+	Command `name:"web" short-description:"Web"`
+}
+
+func (c *webCmd) Execute(args []string) error {
+	if err := <-OpenUI(10 * time.Second); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func init() {
+	rootCmd.AddCommand(&webCmd{})
+}
+
+func openUI() error {
+	var stdout, stderr bytes.Buffer
+	for {
+		if err := compose.RunWithIO(context.Background(),
+			os.Stdin, &stdout, &stderr, "port", "superset", "8088"); err == nil {
+			break
+		}
+
+		time.Sleep(1 * time.Second)
```

Comment:
I'm not sure we want to have this loop retrying.
If the containers are stopped, returning here with a message along the lines of "could not get superset port, is it running?" would be more clear than returning after 10s with "opening the ui took more than"

Reply:
it makes sense to keep loop for `install` command.
depends on the hardware and other loads on the host system it can take a while.